src/views/2_SessionComponent.tsx

```diff
@@ -25,6 +25,10 @@ export class SessionComponent extends React.Component<Props, {}> {
     }
 
     componentDidMount() {
+        this.session.onscroll = () => {
+            const shutterOffset = this.session.scrollTop;
+            this.shutter.style.transform = `translateY(${shutterOffset}px)`;
+        };
```

Comment:
Fixed it in this commit. Closing. https://github.com/shockone/black-screen/commit/862ae28ee1204b12ac91a11ecf35eb15e9aedb95